scripts/check-rust-version.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Define the expected Rust version
+EXPECTED_VERSION="1.75"
```

Comment:
Question: instead of defining the version here, is it possible to read it from the `rust-toolchain` file?

Reply:
Yes it is possible, updated the script to do so.